src/qtscript.cpp

```diff
@@ -303,19 +308,25 @@ Vector2i getPlayerStartPosition(int player)
 	return positions[player];
 }
 
+static nlohmann::json derrickPositions;
```

Comment:
Should this variable be cleared somehow like derricks in `scriptInit`?
https://github.com/Warzone2100/warzone2100/blob/e88d2a478304e540fa4f25bd3b104468d382348c/src/qtscript.cpp#L301

Reply:
most probably..Like between when the map changes